docs/connectors/rhoas-cli-getting-started-connectors/README.adoc

```diff
@@ -102,17 +102,21 @@ For the example in this guide, you create a source connector that sends data fro
 
 Use this guide to complete the following tasks:
 
-* {base-url}{getting-started-rhoas-cli-url-connectors}#proc-create-connector-namespace_connectors-rhoas-cli[Create a namespace to host your {product-connectors} instances]
-* {base-url}{getting-started-rhoas-cli-url-connectors}#proc-building-connector-configuration-cli_connectors-rhoas-cli[Build a configuration file for each type of connector that you want to create]
-* {base-url}{getting-started-rhoas-cli-url-connectors}#proc-create-connector-instances_connectors-rhoas-cli[Create {connectors} instances]
+* {base-url}{getting-started-rhoas-cli-url-connectors}#_75aa250f-388e-4a3c-86b6-d4dc59741df1[Create a namespace to host your {product-connectors} instances]
```

Comment:
This change will break links upstream. Downstream links are already broken: https://access.redhat.com/documentation/en-us/openshift_connectors/1/guide/8da19b11-0ae0-469b-adfd-9bbf9973b30a
 
The broken URLs show an unresolved attribute: 
https://access.redhat.com/documentation/en-us/{getting-started-rhoas-cli-url-connectors}#proc-create-connector-namespace_connectors-rhoas-cli 
 
Looks like you just need to set the `{getting-started-rhoas-cli-url-connectors}` attribute in the downstream RHOC GitLab repo. Seems to be missing, which would explain the broken downstream links. 

BTW, separate issue:  also noticed that all RHOC guides are missing from the top-level upstream `readme.adoc`:  https://github.com/redhat-developer/app-services-guides

Reply:
I will add the attribute downstream